### PR TITLE
feat: create login page F&B

### PR DESCRIPTION
**Pull request by nadiia-tsytsylina**
## Describe your changes

Create a Login page, LoginForm component.
Add styles, validation, show password button, links to reset password page and signup page

Снимок экрана 2023-11-10 181940
Снимок экрана 2023-11-10 181956
Снимок экрана 2023-11-10 182008

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/AC-9)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [ ] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [ ] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [ ] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Comment by nadiia-tsytsylina** [comment]
Fix: update input styles
Снимок экрана 2023-11-13 182119
Снимок экрана 2023-11-13 182136

File: src/components/login-form/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    resolver: yupResolver(loginSchema),
+    mode: 'onBlur',
+  });
+
+  return (
+    <>
+      <Title>Sign in to get access to your appointments</Title>
+      <StyledForm
+        onSubmit={handleSubmit((data) => {
+          alert(JSON.stringify(data));
```
Comment (lyudad) [inline src/components/login-form/LoginForm.tsx:46]: плохое решение мы можем что то не такой в лоб поставить. toast, попап, текст, я алерты ктоме как тестовых проектах нигде не видела

File: src/pages/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Head from 'next/head';
+import LoginWrapper from 'src/components/login-form/LoginWrapper';
+import SignUpHeader from 'src/components/reusable/header';
+import LoginForm from 'src/components/login-form/LoginForm';
+
+export default function Login(): JSX.Element {
+  return (
+    <>
+      <Head>
+        <title>Login page</title>
```
Comment (lyudad) [inline src/pages/login/index.tsx:11]: вижно по чеклисту что не проверила, не знаю почему. чеклист для того что бы проверять, я его знаю на память и ожидаю если не того же то хотя бы открыть и пройтись по нему

**Comment by nadiia-tsytsylina** [comment]
update F&B functionality
add signIn endpoint to authApi.
update submit logic;


image

**Comment by lyudad** [comment]
Screenshot 2023-11-16 at 20 10 58
Screenshot 2023-11-16 at 20 11 23

**Comment by nadiia-tsytsylina** [comment]
именно тесты не писала. убрала галочку

File: .env.example [hunk 3]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=http://localhost:3001
```
Comment (lyudad) [inline .env.example:1]: почему я и в https://github.com/ZenBit-Tech/ctrlchamps_fe/pull/18 и в 9 вижу одни и теже изменения? все пры должны делаться от ветки девелоп

**Review by lyudad — APPROVED** [review]

**Pull request merged**